integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -32,11 +37,11 @@
         'spark.rapids.sql.python.gpu.enabled': 'true'
         }
 
+
 def _create_df(spark):
-    return spark.createDataFrame(
-            [(1, 1.0), (1, 2.0), (2, 3.0), (2, 5.0), (2, 10.0)],
-            ("id", "v")
-        )
+    elements = list(map(lambda i: (i, i/1.0), range(1, 5000)))
```

Comment:
@firestarman What is the follow on issue to fix the underlying problem?  This was a workaround to not expose the bug, but I would much rather see us xfail in that situation pointing at the bug for us to track and fix it.

Reply:
This is tracked by https://github.com/NVIDIA/spark-rapids/issues/750